addons/arsenal/functions/fnc_fillRightPanel.sqf

```diff
@@ -313,9 +313,9 @@ switch (_ctrlIDC) do {
         // Unknown items
         {
             if !(_x in _items) then {
-                ["CfgWeapons", _x, !(_x in GVAR(virtualItemsFlat)), true] call _fnc_fillRightContainer;
+                ["CfgWeapons", _x, _y, true] call _fnc_fillRightContainer;
```

Comment:
Should add comment on why this specifically uses `_y`

Reply:
Done - please review again.